sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -182,40 +216,76 @@ for await (let listedCertificate of client.listCertificates()) {
 }
 ```
 
-### Update a certificate
+### Updating a certificate
 
-`updateCertificate` updates the attributes of a certificate.
+The certificate attributes can be updated to an existing certificate version with
+`updateCertificate`, as follows:
 
 ```javascript
-const result = getCertificate(certificateName, "");
+const result = client.getCertificateWithPolicy(certificateName);
 await client.updateCertificate(certificateName, result.version, {
+  certificatePolicy: {
+    issuerParameters: { name: "Self" }
+  },
+  certificateAttributes: {
+    enabled: false
+  },
   tags: {
-    customTag: "value"
+    myCustomTag: "myCustomTagsValue"
   }
 });
 ```
 
-### Delete a certificate
+The certificate's policy can also be updated individually with `updateCertificatePolicy`, as follows:
+
+```javascript
+const result = client.getCertificateWithPolicy(certificateName);
+await client.updateCertificatePolicy(certificateName, {
+  issuerParameters: { name: "Self" }
+});
+```
+
+### Deleting a certificate
+
+The `deleteCertificate` method sets a certificate up for deletion. This process will
+happen in the background as soon as the necessary resources are available.
+
+```javascript
+await client.deleteCertificate(certificateName);
+```
 
-`deleteCertificate` deletes a certificate previously stored in the Key Vault. When [soft-delete](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-ovw-soft-delete) is not enabled for the Key Vault, this operation permanently deletes the deletes.
+If [soft-delete](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-ovw-soft-delete)
+is enabled for the Key Vault, this operation will only label the certificate as a
+_deleted_ certificate. A deleted certificate can't be updated. They can only be either
+read, recovered or purged.
 
 ```javascript
 await client.deleteCertificate(certificateName);
+
+// If soft-delete is enabled, we can eventually do:
+await client.getDeletedCertificate(certificateName);
+// Deleted certificates can also be recovered or purged:
+await client.recoverDeletedCertificate(certificateName);
+// await client.purgeDeletedCertificate(certificateName);
 ```
 
+Since the deletion of a certificate won't happen instantly, some time is needed
+after the `deleteCertificate` method is called before the deleted certificate is
+available to be read, recovered or purged.
+
 ## Troubleshooting
 
 ### Enable logs
 
 You can set the following environment variable to get the debug logs when using this library.
 
-- Getting debug logs from the KeyVault Certificates SDK
+- Getting debug logs from the Key Vault Certificates SDK
 
 ```bash
 export DEBUG=azure*
 ```
 
-- Getting debug logs from the KeyVault Certificates SDK and the protocol level library.
+- Getting debug logs from the Key Vault Certificates SDK and the protocol level library.
 
 ```bash
 export DEBUG=azure*,rhea*
```

Comment:
Why do we still have this section on logging? As mentioned in #5088, use of the `debug` module for logging is not applicable here. Am guessing this got copied over from event hubs or service bus which do indeed use the `debug` module for logging and so can get the logs this way

Reply:
Yup, I missed this. Cleaning this up.